scripts/check-airspaces

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python
```

Comment:
is this script being used anywhere yet?

Reply:
Not yet. It should be used in the ci chain to check the airspace files for validity.